examples/nl-core-Mobility-01.xml

```diff
@@ -79,84 +45,29 @@
   <note>
     <text value="Rollator is eigen bezit."/>
   </note>
-  <component>
-    <code>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="282097004"/>
-        <display value="loopvermogen"/>
-      </coding>
-    </code>
-    <valueCodeableConcept>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="16581000146103"/>
-        <display value="Needs help with walking"/>
-      </coding>
-    </valueCodeableConcept>
-  </component>
-  <component>
-    <code>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="301587001"/>
-        <display value="vermogen tot traplopen"/>
-      </coding>
-    </code>
-    <valueCodeableConcept>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="301589003"/>
-        <display value="Niet in staat trap te lopen"/>
-      </coding>
-    </valueCodeableConcept>
-  </component>
-  <component>
-    <code>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="282869009"/>
-        <display value="vermogen om van houding te veranderen"/>
-      </coding>
-    </code>
-    <valueCodeableConcept>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="19821000146105"/>
-        <display value="Independently able to change position"/>
-      </coding>
-    </valueCodeableConcept>
-  </component>
-  <component>
-    <code>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="249868004"/>
-        <display value="vermogen om positie aan te houden"/>
-      </coding>
-    </code>
-    <valueCodeableConcept>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="19811000146100"/>
-        <display value="Independently able to maintain position"/>
-      </coding>
-    </valueCodeableConcept>
-  </component>
-  <component>
-    <code>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="364666007"/>
-        <display value="vermogen om te verplaatsen"/>
-      </coding>
-    </code>
-    <valueCodeableConcept>
-      <coding>
-        <system value="http://snomed.info/sct"/>
-        <code value="719024002"/>
-        <display value="Able to transfer location with assistance"/>
-      </coding>
-    </valueCodeableConcept>
-  </component>
+  <hasMember>

```

Comment:
Deze van .component -> .hasMember wijziging ook nog in het ticket opnemen?

Reply:
ZIBFHIR-220 aangemaakt